draftlogs/5854_fix.md

```diff
@@ -0,0 +1 @@
+- Skip `"hoverinfo": "none"` traces for `x/y unified` [[#5854](https://github.com/plotly/plotly.js/pull/5854)]
```

Comment:
The fix proposed in this PR is not specific to "(x|y) unified" `hovermode` and covers other hover modes namely "x" and "y". If so, then please update the log.

Reply:
I did. Thanks